walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +273,130 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
+
+  @Test
+  public void ignoreOtionalWithWholeDayMeeting() {
+    // Have an optional attendee who has an all day meeting, so the meeting is sceduled without him.
+
+    Collection<Event> events = Arrays.asList(
+        new Event("Event 1", TimeRange.fromStartDuration(TIME_0800AM, DURATION_30_MINUTES),
+            Arrays.asList(PERSON_A)),
+        new Event("Event 2", TimeRange.fromStartDuration(TIME_0900AM, DURATION_30_MINUTES),
+            Arrays.asList(PERSON_B)),
+        new Event("Event 3", TimeRange.WHOLE_DAY,
+            Arrays.asList(PERSON_C)));
+
+    MeetingRequest request =
+            new MeetingRequest(Arrays.asList(PERSON_A, PERSON_B), DURATION_30_MINUTES);
+    request.addOptionalAttendee(PERSON_C);
+
+    Collection<TimeRange> actual = query.query(events, request);
```

Comment:
I'm sure they've been over this, but -

In tests we divide the test to three sections: arrange, act, assert. 
Usually we'd want a visual separation of the three blocks. 

Here, that means that we'd separate the line 
`    Collection<TimeRange> actual = query.query(events, request);`
from other lines by adding a newline before and after it.

Reply:
They haven't been over this :) 
The action I should take from this comment is only to add a newline after the actual? 
My style is the same as the tests that they wrote, should I do this for all tests, including those I didn't touch?

Reply:
I sent you an email with the explanation about unit test structure (arrange/act/assert). 

Don't change tests you didn't write (although it's good practice to make sure the entire file is aligned with the same styling!). 

Basically, a new line is what you should take from this comment. 
Taking this a step further would be to add comments in each section ("// Arrange", "// Act", etc.).

Reply:
Done